initfiles/sbin/hpcc-run.sh.in

```diff
@@ -52,134 +52,143 @@ getIPS(){
 }
 
 getDali(){
-        DIP=`${INSTALL_DIR}/sbin/configgen -env ${envfile} -listall | grep Dali | awk -F, '{print \$3}'  | sort | uniq`
+    DIP=`${INSTALL_DIR}/sbin/configgen -env ${envfile} -listall | grep Dali | awk -F, '{print \$3}'  | sort | uniq`
 }
 
 createIPListFile(){
-    _file=$1
-    echo "$IPS" > $_file
+    local _output=$1
+    echo "$IPS" > $_output
 }
 
 createIPListFileExcludeDIP(){
-  _file=$1
-  echo "$IPS" | grep -v $DIP  > $_file
+    local _input=$1
+    local _output=$2
```

Comment:
We may not care since n is typically small, but this is an n^2 process (and I never like them).

I wonder if changing the grep to match "^$DIP\$" (or maybe "^ *$DIP *\$") would have been a simpler fix.